source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -520,14 +523,15 @@ export class TextBlock extends CardElement {
                 element.style.wordWrap = "break-word";
 
                 if (this.maxLines > 0) {
-                    element.style.maxHeight = (computedLineHeight * this.maxLines) + "px";
+                    element.style.maxHeight = (this.computedLineHeight * this.maxLines) + "px";
                     element.style.overflow = "hidden";
                 }
             }
             else {
                 element.style.whiteSpace = "nowrap";
             }
 
+            this.innerHtml = element.innerHTML;
```

Comment:
This shouldn't be necessary. Once a card element is rendered, its renderedElement property is set to the element returned by internalRender. So in updateLayout, you should simply be able to refer to this.renderedElement ansd you shouldn't need this private innerHtml member.

Reply:
Since `this.renderedElement` is a reference to the element, after the truncation code executes, `this.renderedElement.innerHTML` will reflect the truncated content. If the card layout then changes such that the available room for the TextBlock has increased, we need a way to bring the original content back.

Reply:
Makes sense. Maybe a better name would be _nonTruncatedInnerHtml or _originalInnerHtml or something of the sort, but that's a nit pick.